packages/okta-adapter/src/filters/default_rule_deployment.ts

```diff
@@ -126,17 +150,39 @@ const deployDefaultPolicy = async (
 const filterCreator: FilterCreator = ({ definitions, oldApiDefinitions }) => ({
   name: 'defaultPolicyRuleDeployment',
   preDeploy: async changes => {
+    // Get the priority of the default MultifactorEnrollmentPolicy
+    await awu(changes)
+      .filter(isInstanceChange)
+      .filter(isModificationChange)
+      .filter(
+        change =>
+          getChangeData(change).elemID.typeName === MFA_POLICY_TYPE_NAME && getChangeData(change).value.system === true,
+      )
+      .map(change => getChangeData(change))
+      .forEach(async instance => {
+        const response = await definitions.clients.options.main.httpClient.get({
+          url: `/api/v1/policies/${instance.value.id}`,
+        })
+        if (isPolicyResponse(response)) {
+          instance.value.priority = response.data?.priority
```

Comment:
nit: I think `?` is redundant as you already validated the schema
```suggestion
          instance.value.priority = response.data.priority
```